aws-redshiftserverless-namespace/src/main/java/software/amazon/redshiftserverless/namespace/UpdateHandler.java

```diff
@@ -241,13 +242,19 @@ private DeleteResourcePolicyResponse deleteNamespaceResourcePolicy(
         return deleteResponse;
     }
 
-
     private Map<String, software.amazon.awssdk.services.redshiftserverless.model.SnapshotCopyConfiguration> getSnapshotCopyConfigurations(final ProxyClient<RedshiftServerlessClient> proxyClient, ResourceModel model) {
-        ListSnapshotCopyConfigurationsResponse listResponse = proxyClient.injectCredentialsAndInvokeV2(Translator.translateToListSnapshotCopyConfigurationsRequest(model),
-                proxyClient.client()::listSnapshotCopyConfigurations);
-
-        return listResponse.snapshotCopyConfigurations().stream()
-                .collect(Collectors.toMap(software.amazon.awssdk.services.redshiftserverless.model.SnapshotCopyConfiguration::destinationRegion, Function.identity()));
+        try {
+            ListSnapshotCopyConfigurationsResponse listResponse = proxyClient.injectCredentialsAndInvokeV2(Translator.translateToListSnapshotCopyConfigurationsRequest(model),
+                    proxyClient.client()::listSnapshotCopyConfigurations);
+            return listResponse.snapshotCopyConfigurations().stream()
+                    .collect(Collectors.toMap(software.amazon.awssdk.services.redshiftserverless.model.SnapshotCopyConfiguration::destinationRegion, Function.identity()));
+        } catch (Exception ex) {
+            if (ex instanceof ValidationException) {
```

Comment:
Are we sure that all ValidationExceptions mean that feature is not supported. What about if customer IAM permission issue

Reply:
> Are we sure that all ValidationExceptions mean that feature is not supported

Yes, we only throw a ValidationException in the API if there's no access. 

> What about if customer IAM permission issue?
This will throw a AccessDeniedException